deploy-board/deploy_board/templates/deploys/deploy_progress.tmpl

```diff
@@ -13,29 +13,69 @@
             {% for agentStat in report.agentStats %}
                 {% for hostTag in hostTags %}
                     {% if agentStat.agent.hostName == hostTag.hostName %}
+                        {% if report.showMode == "complete" or report.showMode == "compact" %}
                         <a href="/env/{{env.envName}}/{{env.stageName}}/host/{{ agentStat.agent.hostName }}"
-                           type="button" class="deployToolTip btn btn-xs {{ agentStat | agentButton }} host-btn"
-                           title="{{ agentStat | agentTip }}">
-                           {% if report.showMode == "complete" %}
-                           <small>{{ agentStat.agent.hostName }}</small>
-                           {% endif %}
-                           <i class="fa fa-fw {{ agentStat | agentIcon }}"></i>
+                        type="button" class="deployToolTip btn btn-xs {{ agentStat | agentButton }} host-btn"
+                        title="{{ agentStat | agentTip }}">
+                        {% if report.showMode != "compact" %}
+                        <small>{{ agentStat.agent.hostName }}</small>
+                        {% endif %}
+                        <i class="fa fa-fw {{ agentStat | agentIcon }}"></i>
+                        </a>
+                        {% elif report.showMode == "main" %}
+                        {% if not agentStat.agent.accountId or agentStat.agent.accountId == 998131032990 %}
```

Comment:
let's have a primaryAccount flag for each agent in py code.

Reply:
sounds reasonable, thanks